### PR TITLE
Adding organism for PageDetails, Adding DateModified Atom to include …

### PR DESCRIPTION
**Pull request by P0NDER0SA**
…in the page Details, fixed some linting, included unit tests and included the axe test framework

# Description

Adding the date modified and skeleton for the PageDetails.  included unit tests for the date modified.  included the Axe testing framework.  updated the environment variables in the configuration to default and format values accordingly.

1-3 sentence description of the feature you added.

## Test Instructions
run the app, look for the date modified information above the footer.  should be "DATE-IS-NA" in dev (unless .env variable is supplied), should be date of Team City build when deployed to cloud.

## Meets Definition of Done

- [ X] Meets design spec
- [ X] unit tests are included

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
-import Link from "next/link";
-import Layout, { siteTitle } from "../components/layout";
+import PageDetails from "../components/organisms/PageDetails";
+import Layout from "../components/layout";
 
 export default function Home() {
-  return <Layout home></Layout>;
+  return (
+    <Layout home>
+      <PageDetails />
```
Comment (krischarbonneau) [inline pages/index.js:7]: Wouldn't this be better to have in the layout above the footer?

Comment (P0NDER0SA) [reply pages/index.js:7]: I agree!

**Review by krischarbonneau — APPROVED** [review]
Looks good, great work Pond!

File: components/atoms/DateModified.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Contains build time stamp
+ */
+export default function DateModified(props) {
+  return (
+    <div property="dateModified">
+      Date Modified: {process.env.NEXT_PUBLIC_BUILD_DATE}
```
Comment (krischarbonneau) [inline components/atoms/DateModified.js:7]: Just noticed this after approving, insignificant but the first letter of Modified should not be capitalized in order to match what is on Canada.ca

There should also be 2em of padding on top and bottom, text should be Noto Sans, text color should be #333, font-size should be 16px, line-height should be 1.65em

**Pull request merged**